### ISSUE TITLE
"Statustext" is wrong word. "Signature" would be better

### ISSUE THREAD

**Issue by angelo-fuchs** [deltachat/deltachat-android-ii#247]
In "Settings" -> "Profile"
The label for the second text field is "Statustext" but the field does not denote a "Status" but instead is the "Signature" that is shown to users of non-delta mail clients.

It should instead be called "Mail Signature" or something similar.

**Comment by angelo-fuchs** [comment]
Additionally the text in "Who can see this" does not explain what the text field is and who will be seeing it.

**Comment by r10s** [comment]
@violoncelloCH yes, this was the initial idea, and i would still like to see the text also there, however, it's not the current state, so i think it is worth changing the description.

**Comment by violoncelloCH** [comment]
@r10s or we indicate in the text that this will change in the future / the feature will come...
